src/build.config.js

```diff
@@ -22,7 +22,8 @@ export default {
       'ARBETH',
       'FISH',
       'LUNA',
-      'UST'
+      'UST',
+      'FUSE'
```

Comment:
Please remove fuse from the default assets,
only assets which we have vetted very closely are added to the user's wallet by default
Users can still use fuse by enabling it in Settings -> Manage assets